src/main/java/pw/kaboom/extras/modules/checks/DenyMobLag.java

```diff
@@ -0,0 +1,24 @@
+package pw.kaboom.extras.modules.checks;
+
+import org.bukkit.entity.EntityType;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.CreatureSpawnEvent;
+import org.bukkit.plugin.java.JavaPlugin;
+import pw.kaboom.extras.Main;
+
+public class DenyMobLag implements Listener {

```

Comment:
Extras already has a mob limiter. Consider removing this class or merging it into the current mob limiter implementation.